### PR TITLE
Replace lion logo with new one

### PR DESCRIPTION
**Pull request by malindamiller**
I made a logo using the Lion Ambassador Logo because I felt like we should perhaps incorporate that.  Let me know what you think.

image

### CODE REVIEW

**Comment by austinkelleher** [comment]
:+1:

File: behrend-lion-ambassadors/app/views/layouts/application.html.erb [hunk 1]
```diff
@@ -31,7 +31,7 @@
 		<!-- image-container -->
 		<div id="image-container">
 
-			<%= image_tag("logo-new.png") %>	<!-- rails for showing image -->
+			<%= image_tag("liologo4.png") %>	<!-- rails for showing image -->
```
Comment (austinkelleher) [inline behrend-lion-ambassadors/app/views/layouts/application.html.erb:34]: Consider renaming this and removing the duplicate image.

File: behrend-lion-ambassadors/config/initializers/assets.rb [hunk 2]
```diff
@@ -24,3 +24,4 @@
 Rails.application.config.assets.precompile += %w( carousel4.jpg )
 Rails.application.config.assets.precompile += %w( emails.css )
 Rails.application.config.assets.precompile += %w( events.css )
+Rails.application.config.assets.precompile += %w( liologo4.png )
```
Comment (austinkelleher) [inline behrend-lion-ambassadors/config/initializers/assets.rb:27]: Same as above comment.

**Comment by malindamiller** [comment]
I merged this.  We can always change it again if you disagree

**Pull request merged**